tests/sentinel/tests/10-replica-priority.tcl

```diff
@@ -34,6 +34,9 @@ proc 10_test_number_of_replicas {n_replicas_expected} {
         # ensure sentinels replies with the right number of replicas
         foreach_sentinel_id id {
             # retries 40 x 500ms = 20s as SENTINEL_INFO_PERIOD = 10s
+            S $id sentinel debug info-period 100
+            S $id sentinel debug default-down-after 1000
+            S $id sentinel debug publish-period 100
             set len [llength [S $id SENTINEL REPLICAS mymaster]]
             wait_for_condition 40 500 {
```

Comment:
Here we already decease info-period to 0.1 second, if wait_for_condition parameter could be decreased?